sdk/tests/js/webgl-test-utils.js

```diff
@@ -2015,6 +2015,19 @@ var loadShaderFromFile = function(
       opt_logShaders, undefined, file, opt_skipCompileStatus);
 };
 
+var loadShaderFromFileAsync = function(
+    gl, file, type, opt_errorCallback, opt_logShaders, opt_skipCompileStatus, callback) {
+  loadTextFileAsync(file, function(gl, type, opt_errorCallback, opt_logShaders, file, opt_skipCompileStatus){
+      return function(success, shaderSource) {
+        if (success) {
+          var shader = loadShader(gl, shaderSource, type, opt_errorCallback,
+              opt_logShaders, undefined, file, opt_skipCompileStatus);
+          callback(true, shader);
```

Comment:
In the error case, "callback" isn't called with a false first argument. This will cause callers to hang in the case of an error. Could you please make the code more robust by calling:
  callback(false, null);
in the error case?